src/app/modules/item/components/chapter-children/chapter-children.component.ts

```diff
@@ -7,14 +8,15 @@ import { ItemData } from '../../services/item-datasource.service';
   templateUrl: './chapter-children.component.html',
   styleUrls: ['./chapter-children.component.scss'],
 })
-export class ChapterChildrenComponent implements OnChanges {
+export class ChapterChildrenComponent implements OnChanges, OnDestroy {
   @Input() itemData: ItemData;
 
   state: 'loading' | 'ready' | 'error' = 'loading';
   children: ItemChild[] = [];
 
   constructor(private getItemChildrenService: GetItemChildrenService) {}
 
+  private subscription: Subscription;
```

Comment:
as it does hold any value after definition and constructor, make is optional. (and handle the optional in ngOnDestroy)
```suggestion
  private subscription?: Subscription;
```

Reply:
fixed